dspace-api/src/main/java/org/dspace/handle/HandleClarinServiceImpl.java

```diff
@@ -109,6 +109,21 @@ public Handle findByHandle(Context context, String handle) throws SQLException {
         return handleDAO.findByHandle(context, handle);
     }
 
+    @Override
+    public Handle createExternalHandle(Context context, String handleStr, String url, Boolean dead, Date deadSince)
+            throws SQLException, AuthorizeException {
+        // Check authorisation: Only admins may create DC types
+        if (!authorizeService.isAdmin(context)) {
+            throw new AuthorizeException(
+                    "Only administrators may modify the handle registry");
```

Comment:
Why the handle registry and not the external handle?

Reply:
Yes, this comment is too general. I will use something more specific to the problem.